ktor-server/ktor-server-cio/jvmAndNix/src/io/ktor/server/cio/EngineMain.kt

```diff
@@ -19,8 +19,12 @@ public object EngineMain {
     public fun main(args: Array<String>) {
         val applicationEnvironment = commandLineEnvironment(args)
         val engine = CIOApplicationEngine(applicationEnvironment) { loadConfiguration(applicationEnvironment.config) }
+        val gracePeriod =
+            engine.environment.config.propertyOrNull("ktor.deployment.gracePeriod")?.getString()?.toLong() ?: 50
+        val timeout =
+            engine.environment.config.propertyOrNull("ktor.deployment.timeout")?.getString()?.toLong() ?: 5000
```

Comment:
`ktor.deployment.timeout` is to generic name. can you specify that it's the timeout for stopping a server? same for `gracePeriod`